### PR TITLE
Dmp 2694 arm batchsize db driven

### PR DESCRIPTION
**Pull request by garethmorriscgi**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DMP-2694 

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/resources/db/migration/common/V1_298__alter_automated_task_table.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+ALTER TABLE automated_task ADD COLUMN batch_size INTEGER;
```
Comment (karen-hedges) [inline src/main/resources/db/migration/common/V1_298__alter_automated_task_table.sql:1]: Should this be set with a default value of 0 or 1?

File: src/integrationTest/java/uk/gov/hmcts/darts/task/service/AutomatedTaskServiceTest.java [hunk 2]
```diff
@@ -96,10 +95,10 @@ class AutomatedTaskServiceTest extends IntegrationPerClassBase {
     private UnstructuredAudioDeleterProcessor unstructuredAudioDeleterProcessor;
 
     @Autowired
-    private UnstructuredToArmProcessor unstructuredToArmProcessor;
+    private AutomatedTaskProcessorFactory unstructuredToArmProcessor;
 
     @Autowired
-    private ArmResponseFilesProcessor armResponseFilesProcessor;
+    private AutomatedTaskProcessorFactory processorFactory;
```
Comment (johnrace0) [inline src/integrationTest/java/uk/gov/hmcts/darts/task/service/AutomatedTaskServiceTest.java:101]: maybe rename this to indicate it refers to arm response files? if thats the case :-)

Comment (garethmorriscgi) [reply src/integrationTest/java/uk/gov/hmcts/darts/task/service/AutomatedTaskServiceTest.java]: Updated name to armResponseFilesProcessor

**Review by johnrace0 — APPROVED** [review]
happy to approve this since already reviewed by KH

**Pull request merged**